test/BandCollection.ts

```diff
@@ -0,0 +1,28 @@
+import { Type } from 'class-transformer';
```

Comment:
Instead of using Type from `class-transformer` is better to the re-exported Type from this library (as it was before) so it actually breaks if something changes.